code/procs/jobprocs.dm

```diff
@@ -471,6 +512,25 @@
 				B.badge_owner_name = src.real_name
 				B.badge_owner_job = src.job
 
+	if (src.traitHolder && src.traitHolder.hasTrait("pilot"))
+		var/obj/item/tank/emergency_oxygen/E = new /obj/item/tank/emergency_oxygen(src)
+		src.equip_if_possible(E, slot_in_backpack)
+		#ifdef UNDERWATER_MAP
+		var/obj/item/clothing/suit/space/diving/civilian/SSW = new /obj/item/clothing/suit/space/diving/civilian(src)
+		src.equip_if_possible(SSW, slot_in_backpack)
+		var/obj/item/clothing/head/helmet/space/engineer/diving/civilian/SHW = new /obj/item/clothing/head/helmet/space/engineer/diving/civilian(src)
+		src.equip_if_possible(SHW, slot_in_backpack)
+		#else
+		var/obj/item/clothing/suit/space/emerg/SSS = new /obj/item/clothing/suit/space/emerg(src)
+		src.equip_if_possible(SSS, slot_in_backpack)
+		var/obj/item/clothing/head/emerg/SHS = new /obj/item/clothing/head/emerg(src)
+		src.equip_if_possible(SHS, slot_in_backpack)
+		#endif
+		var/obj/item/clothing/mask/breath/MSK = new /obj/item/clothing/mask/breath(src)
+		src.equip_if_possible(MSK, slot_in_backpack)
+		var/obj/item/device/gps/GPSDEVICE = new /obj/item/device/gps(src)
+		src.equip_if_possible(GPSDEVICE, slot_in_backpack)
```

Comment:
Note that if there isn't enough space in the backpack the items will stay sort of "implanted" in you. I recommend doing
`new /obj/item/clothing/mask/breath(src.loc)`
instead of
`new /obj/item/clothing/mask/breath(src)`
etc.
So the leftovers that didn't fit will instead be in the pod (and will spill out when the pilot exits it).

Reply:
Done.